sdk/communication/Azure.Communication.Chat/src/ChatClientOptions.cs

```diff
@@ -25,7 +25,7 @@ public ChatClientOptions(ServiceVersion version = LatestVersion)
         {
             ApiVersion = version switch
             {
-                ServiceVersion.V2021_04_05_preview6 => "2021-04-05-preview6",
+                ServiceVersion.V2021_09_07 => "2021-09-07",
```

Comment:
We normally allow for older versions to be specified.   Is there a reason that we're disallowing that here and, if so, have we confirmed that it won't be viewed as a breaking change?   @tg-msft,  do you have feedback here?

If we do end up validating against older versions, we probably want to mark the older enum members with `[EditorBrowsable(EditorBrowsableState.Never)]`

Reply:
I was confused on how this should work. Got clarification in the .NET SDK channel. Thanks for pointing this out!

Reply:
We do drop preview versions when we go to GA though (because a GA client library should never talk to a preview service version which is still subject to change).  But otherwise, yes, it's important that our libraries support multiple service versions.